etcdserver/member.go

```diff
@@ -35,7 +35,8 @@ var (
 
 // RaftAttributes represents the raft related attributes of an etcd member.
 type RaftAttributes struct {
-	// TODO(philips): ensure these are URLs
+	// PeerURLs are validated by the function types.NewURLs
+	// if they are valid URLs or not.
```

Comment:
You are right. I will change the comment to `PeerURLs must be valid URLs.`

Thanks